src/sentry/integrations/bitbucket/webhook.py

```diff
@@ -206,6 +262,6 @@ def post(self, request: HttpRequest, organization_id: int) -> HttpResponse:
             domain=IntegrationDomain.SOURCE_CODE_MANAGEMENT,
             provider_key=event_handler.provider,
         ).capture():
-            event_handler(event, organization=organization)
+            event_handler(event, request=request, organization=organization)
```

Comment:
what do we need the `request` for here? it's not being used in `BitbucketWebhook` except to check that it's passed

Reply:
`request` contains `X-Hub-Signature` header and request body which are used here:
https://github.com/getsentry/sentry/blob/90fc14b3889a3e26315d1328a3b49976b13d1934/src/sentry/integrations/bitbucket/webhook.py#L135-L142

Reply:
oh wait, can we do this in the webhook itself before calling the handler?